BodyPartDetection Nose
BodyPartDetection Neck
BodyPartDetection RShoulder
BodyPartDetection RElbow
BodyPartDetection RWrist
BodyPartDetection LShoulder
BodyPartDetection LElbow
BodyPartDetection LWrist
BodyPartDetection RHip
BodyPartDetection RKnee
BodyPartDetection RAnkle
BodyPartDetection LHip
BodyPartDetection LKnee
BodyPartDetection LAnkle
BodyPartDetection REye
BodyPartDetection LEye
BodyPartDetection REar
BodyPartDetection LEar
BodyPartDetection Chest
